torchvision/prototype/datasets/_builtin/cub200.py

```diff
@@ -173,9 +177,8 @@ def _make_datapipe(
             )
 
```

Comment:
A second thought. Could we simply filter `image_files_dp` from `archive_dp` here and create the `image_files_map` dictionary?

Then, we can do demux over `archive_dp` again and drop data in `image_files_dp`.

Reply:
So basically splitting of `image_files_dp` from the graph?

Reply:
Yeah. Then, we can materialize the data from it like a meta-datapipe.